Lesson 1

olexandr-gorkavenko opened the pull request:
Форкнул репозиторий, добавил index.html, создал пулл реквест

olexandr-gorkavenko committed:
Добавил gitignore

olexandr-gorkavenko committed:
Добавил index.html

olexandr-gorkavenko committed:
add task1

olexandr-gorkavenko committed:
add task2

olexandr-gorkavenko committed:
add task3

olexandr-gorkavenko committed:
add task4

olexandr-gorkavenko commented:
Add homework tasks 1-4.

olexandr-gorkavenko committed:
In the file task3.js added an output result array named out.

andrey-shostik commented on lesson-1/task1.js:
```diff
@@ -0,0 +1,17 @@
+
+const arr = [7, 5, 23, 11, 18, 1, 8, 3, 'text', undefined];
+
+function getSumArray() {
```
 массив должен браться из параметров функции

andrey-shostik commented on lesson-1/task3.js:
```diff
@@ -0,0 +1,23 @@
+        uniqArr.push(obj[key]);
+    }
+
+    return uniqArr;
```
http://prntscr.com/j31gsl

andrey-shostik commented on lesson-1/task2.js:
```diff
@@ -0,0 +1,38 @@
+// Задание 2 способ 1, через метод sort
+
+function compareNumeric(a, b) {
+    if (a > b) return 1;
```
https://learn.javascript.ru/coding-style
http://prntscr.com/j31f9z

olexandr-gorkavenko replied:
согласен

andrey-shostik commented on index.html:
```diff
@@ -0,0 +1,10 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```
надо подключать скрипты на страницу

olexandr-gorkavenko replied:
> надо подключать скрипты на страницу
сделано

olexandr-gorkavenko committed:
Added a page title h1, connected scripts task1.js - task4.js on the page.

olexandr-gorkavenko committed:
For GetSumArray() added parameter arr.

olexandr-gorkavenko committed:
const unSortArr renamed to notUniqArr;
instead of the value of the property (obj[key])
we pass into the uniqArr the name of the property (key).
